teslajsonpy/controller.py

```diff
@@ -988,7 +988,10 @@ async def _get_and_process_energysite_data(energysite_id: Text) -> None:
                     data = None
                 if data and data["response"]:
                     response = data["response"]
-                    if response["grid_status"] == "Active":
+                    if (
+                        "grid_status" not in response
```

Comment:
FYI, another way to write this is to do the get to a variable that you then test. You test it's not None and then test it's value. For example, in the other part of this change you have already gotten 'solar_power' as __solar_power. Just run your test on that to avoid redundant gets.

Reply:
That makes sense but in this situation, should I just use get every time because the response does not have the `grid_status` key if the status is actually `Active` but it does have the key when it's `Unknown`, so it varies even with the same system when the key exists.